openquakeplatform/openquakeplatform/static/js/risk_input_model_exposure.js

```diff
@@ -0,0 +1,428 @@
+/*
+   Copyright (c) 2015, GEM Foundation.
+
+      This program is free software: you can redistribute it and/or modify
+      it under the terms of the GNU Affero General Public License as
+      published by the Free Software Foundation, either version 3 of the
+      License, or (at your option) any later version.
+
+      This program is distributed in the hope that it will be useful,
+      but WITHOUT ANY WARRANTY; without even the implied warranty of
+      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+      GNU Affero General Public License for more details.
+
+      You should have received a copy of the GNU Affero General Public License
+      along with this program.  If not, see <https://www.gnu.org/licenses/agpl.html>.
+*/
+
+var exposureTable;
+var header;
+var showArea = false;
+
+$( document ).ready(function() {
+    updateTable();
+    $('#outputDiv').css('display', 'none');
+});
+
+$('#retrofittingSelect').hide();
+
+/////////////////////////////////////////////////////////
+// Manage the visibility of the perArea selection menu //
+/////////////////////////////////////////////////////////
+$('#perArea').hide();
+
+$('#defineCostStruc').change(function() {
+    defineCost($(this).val());
+    if (defineCost != 'none') {
```

Comment:
If you select Structural Costs different than "None", the "Retrofitting Cost" checkbox is shown. But if afterwards you select "None" again, the checkbox doesn't disappear. I believe substituting `defineCost` with `$(this).val()` at line 36, it will work correctly.